client/lib/signup/step-actions.js

```diff
@@ -179,6 +180,12 @@ module.exports = {
 			var errors = error && error.error ? [ { error: error.error, message: error.message } ] : undefined,
 				bearerToken = error && error.error ? {} : { bearer_token: response.bearer_token };
 
+			if ( ! errors ) {
+				// Fire after a new user registers.
+				analytics.tracks.recordEvent( 'calypso_user_registration_complete' );
```

Comment:
I tried to get these props but didn't find a way. We can always do that in another PR, though. :)